examples/misc/upload_image_assets.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example uploads an image asset.
+
+To get image assets, run get_all_image_assets.py."""
+
+
+import argparse
+import sys
+import requests
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id):
+    """Main method, to run this code example as a standalone application."""
+
+    # Download image from URL
+    URL = "https://goo.gl/3b9Wfh"
+    image_content = requests.get(URL).content
+
+    asset_operation = client.get_type('AssetOperation', version='v2')
+    asset = asset_operation.create
+    asset.type = client.get_type('AssetTypeEnum', version='v2').IMAGE
+    asset.image_asset.data.value = image_content
+    asset.image_asset.file_size.value = len(image_content)
+    asset.image_asset.mime_type = client.get_type('MimeTypeEnum').IMAGE_JPEG
+    # Use your favorite image library to determine dimensions
+    asset.image_asset.full_size.height_pixels.value = 315
+    asset.image_asset.full_size.width_pixels.value = 600
+    asset.image_asset.full_size.url.value = URL
+    # Optional: Provide a unique friendly name to identify your asset.
+    # If you specify the name field, then both the asset name and the image
+    # being uploaded should be unique, and should not match another ACTIVE
+    # asset in this customer account.
+    # asset.name = 'Jupiter Trip #' + ExampleUtilities.GetRandomString(),
+
+    asset_service = client.get_service('AssetService', version='v2')
+
+    try:
+        mutate_asset_response = (
+            asset_service.mutate_assets(customer_id,
+                                        [asset_operation])
+        )
+        print(f'Uploaded file(s):')
```

Comment:
*nit*
This doesn't appear to need to be an f-string.

Reply:
fixed